docs/Spell.md

```diff
@@ -11,1252 +11,260 @@ These values are used by the core and a few spell\_\* tables.
+| 200 | ToolTip_13                    | string |
+| 201 | ToolTip_14                    | string |
+| 202 | ToolTip_15                    | string |
+| 203 | ToolTipFlags                  | uint   |
+| 204 | ManaCostPercentage            | uint   |
+| 205 | StartRecoveryCategory         | uint   |
+| 206 | StartRecoveryTime             | uint   |
+| 207 | MaxTargetLevel                | uint   |
+| 208 | SpellFamilyName               | uint   |
+| 209 | SpellFamilyFlagsLow           | flag96 |
+| 210 | SpellFamilyFlagsHigh          | flag96 |
+| 211 | SpellFamilyFlags2             | flag96 |
+| 212 | MaxAffectedTargets            | uint   |
+| 213 | DmgClass                      | uint   |
+| 214 | PreventionType                | uint   |
+| 215 | StanceBarOrder                | uint   |
+| 216 | EffectDamageMultiplier1       | float  |
+| 217 | EffectDamageMultiplier2       | float  |
+| 218 | EffectDamageMultiplier3       | float  |
+| 219 | MinFactionId                  | uint   |
+| 220 | MinReputation                 | uint   |
+| 221 | RequiredAuraVision            | uint   |
+| 222 | TotemCategory1                | uint   |
+| 223 | TotemCategory2                | uint   |
+| 224 | AreaGroupId                   | int    |
+| 225 | SchoolMask                    | uint   |
+| 226 | runeCostID                    | uint   |
+| 227 | spellMissileID                | uint   |
+| 228 | PowerDisplayId                | uint   |
+| 229 | EffectBonusMultiplier1        | float  |
+| 230 | EffectBonusMultiplier2        | float  |
+| 231 | EffectBonusMultiplier3        | float  |
+| 232 | spellDescriptionVariableID    | uint   |
+| 233 | SpellDifficultyId             | uint   |
 
 ## **Category**
 
-<table>
-<colgroup>
-<col width="50%" />
-<col width="50%" />
-</colgroup>
-<tbody>
-<tr class="odd">
-<td><p><strong>ID</strong></p></td>
-<td><p><strong>Description</strong></p></td>
-</tr>
-<tr class="even">
-<td><p>0</p></td>
-<td><p>Default</p></td>
-</tr>
-<tr class="odd">
-<td><p>1</p></td>
-<td><p>Summon guards</p></td>
-</tr>
-<tr class="even">
-<td><p>2</p></td>
-<td><p>Entry</p></td>
-</tr>
-<tr class="odd">
-<td><p>4</p></td>
-<td><p>Entry</p></td>
-</tr>
-</tbody>
-</table>
+| ID | Description   |
+|----|---------------|
+| 0  | Default       |
+| 1  | Summon guards |
+| 2  | Entry         |
+| 4  | Entry         |
 
 ## **powerType**
 
-<table>
-<colgroup>
-<col width="50%" />
-<col width="50%" />
-</colgroup>
-<tbody>
-<tr class="odd">
-<td><p><strong>ID</strong></p></td>
-<td><p><strong>Description</strong></p></td>
-</tr>
-<tr class="even">
-<td><p>0</p></td>
-<td><p>Mana</p></td>
-</tr>
-<tr class="odd">
-<td><p>1</p></td>
-<td><p>Rage</p></td>
-</tr>
-<tr class="even">
-<td><p>2</p></td>
-<td><p>?</p></td>
-</tr>
-<tr class="odd">
-<td><p>3</p></td>
-<td><p>Energy</p></td>
-</tr>
-</tbody>
-</table>
+| ID | Description |
+|----|-------------|
+| 0  | Mana        |
+| 1  | Rage        |
+| 2  |             |
```

Comment:
in wowdev the powertype value "2" is designed as "Focus", we could use the same or leave blank
https://wowdev.wiki/DB/PowerType

Reply:
Thanks for the information. Maybe I don't pay much attention to those details, because I'm just converting from HTML to Markdown. Then in the next step, the idea is to check to see how outdated we have the tables and at that time, if we need the information that we are missing.